helpers/Upload.js

```diff
@@ -1,18 +1,22 @@
+import Fs from 'fs';
 import Fetch from 'node-fetch';
 
 const upload = async (posts) => {
-  try {
-    await Fetch(`${process.env.SCRAPER_ADMIN_URL}/upload`, {
-      headers: {
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-      },
-      method: 'POST',
-      body: JSON.stringify(posts),
-    });
-    console.log('Upload successfully!');
-  } catch (err) {
-    console.log(err);
+  const scraper = JSON.parse(Fs.readFileSync('./scraper.json', 'utf8'));
```

Comment:
which `scraper.json` file will be read here? should we use this `${process.env.PWD}/scraper.json`?

Reply:
there're 2 places reading this file. so we should create a new `readScraper` function to read this file.